packages/@o3r/dev-tools/src/cli/artifact-cleaner.ts

```diff
@@ -9,8 +9,8 @@ import * as winston from 'winston';
 
 program
   .description('Clean old artifacts from artifactory repositories')
-  .option('--artifactory-url <artifactoryUrl>', 'Artifactory URL', /.*/, 'https://digitalforairlines.jfrog.io/digitalforairlines/')
-  .option('-d, --duration-kept <durationKept>', 'All artifacts which have not been downloaded and are older than this value(ms) will be deleted. Default to 10080000ms (7 days)', /.*/, '604800000')
+  .option('--artifactory-url <artifactoryUrl>', 'Artifactory URL', 'https://digitalforairlines.jfrog.io/digitalforairlines/')
```

Comment:
does it make sense to keep this default value?

Reply:
I did not think much than removing deprecated function call.
If so maybe a clean up of several CLIs options can be done in separated task